### PR TITLE
LRQA 64122 | master

### PR DESCRIPTION
**Pull request by calvinctvo**
https://issues.liferay.com/browse/LRQA-64122

### CODE REVIEW

**Comment by calvinctvo** [comment]
ci:test:sf

**Comment by calvinctvo** [comment]
ci:test:relevant

**Comment by guipfreire** [comment]
ci:test:relevant

**Comment by rodrigocunhaa** [comment]
Just started reviewing :)

File: modules/apps/app-builder/app-builder-test/src/testFunctional/tests/AppBuilder.testcase [hunk 1]
```diff
@@ -590,6 +590,35 @@ definition {
...
+		AppBuilderAdmin.addCustomObject(
+			customObjectName = "my custom object",
+			isCreatingFormView = "true",
+			isCreatingViaPlusIcon = "true");
+
+		FormViewBuilder.addFieldByDoubleClick(fieldType = "Text");
+
+		FormViewBuilder.setFieldLabel(fieldLabel = "Name");
+
+		FormViewBuilder.validateDisabledRequiredField();
```
Comment (rodrigocunhaa) [inline modules/apps/app-builder/app-builder-test/src/testFunctional/tests/AppBuilder.testcase:607]: Hi Calvin, here you have to validate that the kebab menu from "Required Field" is disabled, not the switch button.

File: modules/apps/app-builder/app-builder-test/src/testFunctional/tests/AppBuilder.testcase [hunk 2]
```diff
@@ -590,6 +590,35 @@ definition {
...
+
+		FormViewBuilder.setFieldRequired();
+
+		FormViewBuilder.validateRequiredField();
+
+		FormViewBuilder.validateRequiredIcon();
+
+		FormViewBuilder.validateRequiredIconNotPresent();
```
Comment (rodrigocunhaa) [inline modules/apps/app-builder/app-builder-test/src/testFunctional/tests/AppBuilder.testcase:615]: All you need to do here is validate that the kebab menu from "Required Field" is enabled.

File: modules/apps/app-builder/app-builder-test/src/testFunctional/tests/AppBuilder.testcase [hunk 3]
```diff
@@ -590,6 +590,35 @@ definition {
...
+
```
Comment (rodrigocunhaa) [inline modules/apps/app-builder/app-builder-test/src/testFunctional/tests/AppBuilder.testcase:616]: You are missing one scenario. You need to click on the kebab menu and assert that the "Only for This Form" option is selected.

File: modules/apps/app-builder/app-builder-test/src/testFunctional/tests/AppBuilder.testcase [hunk 4]
```diff
@@ -590,6 +590,35 @@ definition {
...
+		FormViewBuilder.setName(formViewName = "Form Required");
+
+		FormViewBuilder.save();
```
Comment (rodrigocunhaa) [inline modules/apps/app-builder/app-builder-test/src/testFunctional/tests/AppBuilder.testcase:619]: There is no need for these steps, you can remove it.

**Review by rodrigocunhaa — CHANGES_REQUESTED** [review]
Hi @calvinctvo ,

Could you please make the changes that I mentioned and also fix the conflicts?

Thanks!

**Comment by rodrigocunhaa** [comment]
Brian has decided to pause App Builder.

**Pull request closed without merging**